packit/jobs.py

```diff
@@ -81,10 +158,16 @@ def parse_event(
             # we should probably break this method down and move it to handlers or JobTrigger
 
             # github webhooks
-            respone = self.get_package_config_from_github_release(event)
-            if respone:
-                return respone
-            # TODO: pull requests
+            response = self.get_package_config_from_github_release(event)
```

Comment:
The name is a little bit misleading. I know that we have type annotations, but `get_config` looks like it returns an instance of config, not a tuple...

Reply:
ok, I'll rename